assets/src/components/inactiveNotificationModal.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from "react"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import { closeIcon } from "../helpers/icon"
+import { Notification } from "../realtime.d"
+import { setNotification } from "../state"
+import { title } from "./notifications"
+
+const InactiveNotificationModal = ({
+  notification,
+  removeNotification,
+}: {
+  notification: Notification
+  removeNotification: (id: number) => void
+}) => {
+  const [, dispatch] = useContext(StateDispatchContext)
+
+  const closeModal = () => {
+    dispatch(setNotification(undefined))
```

Comment:
I think you also need `dispatch(setSelectedNotificationIsInactive(false))` so the next time you click a notification it doesn't show the modal right away

Reply:
No we're covered there, check out the `selectedNotificationIsInactiveReducer`. Any time a `SET_NOTIFICATION` event is dispatched--including when the notification is `undefined`--we clear the is-inactive flag.